web/src/settings_components.ts

```diff
@@ -1525,3 +1525,31 @@ export function create_group_setting_widget({
 
     return pill_widget;
 }
+
+export function set_expires_on_text($elem: JQuery, valid_to_text: string): void {
+    if ($elem.val() === "custom") {
```

Comment:
Can we be a bit more descriptive with what this jquery element is for these functions ... maybe "$time_select_elem"?